hail/python/cluster-tests/test-skat.py

```diff
@@ -0,0 +1,4 @@
+import hail as hl
+mt = hl.balding_nichols_model(3, 100, 100)
+t = hl.skat(mt.locus, mt.ancestral_af, mt.pop, mt.GT.n_alt_alleles(), covariates=[1])
```

Comment:
Was it not being tested before? Should this go in the main hail tests?

Reply:
We test `skat` in `test_statgen.py`. This cluster test is designed as a sanity check just to make sure the code runs in a cluster environment.